lms/models/lti_params.py

```diff
@@ -26,6 +29,13 @@ def __init__(self, v11: dict, v13: dict | None = None):
     def v11(self):
         return self
 
+    def get_datetime(self, key: str) -> datetime | None:
```

Comment:
We were doing this same thing for course dates.

Generalize it a bit and move it to LTIParams.